help-beta/src/styles/main.css

```diff
@@ -57,6 +57,14 @@
     fill: currentcolor;
 }
 
+.navigation-step-relative-type .zulip-unplugin-icon {
+    /* There's no space between the icon and text for navigation
+    step labels because of the underline text decoration when these
+    steps have a relative link. So we add a right margin to the
+    icon to add the white space without the text formatting.*/
+    margin-right: 4px;
```

Comment:
This `4px` value might be exactly what we want, but it seemed like a good starting place.

Reply:
I can merge this, but we should perhaps brainstorm a bit more in chat.zulip.org on this point.